ote/src/clj/ote/services/pre_notices/attachments.clj

```diff
@@ -49,6 +49,10 @@
               (fn [out]
                 (io/copy (:input-stream s3-file) out)))})))
 
+(defn delete-from-aws [db {bucket :bucket :as config} attachments]
```

Comment:
Maybe delete-from-s3 would be more specific?

Reply:
Fixed. Renamed.